bcda/models/models.go

```diff
@@ -166,20 +166,29 @@ type ACO struct {
 	ACOBeneficiaries []*ACOBeneficiary
 }
 
-func (aco *ACO) GetBeneficiaryIDs() (beneficiaryIDs []string, err error) {
+func (aco *ACO) GetBeneficiaryIDs() (cclfBeneficiaryIDs []string, err error) {
+	if aco.CMSID == nil {
+		log.Errorf("No CMSID set for ACO: %s", aco.UUID)
+		return cclfBeneficiaryIDs, fmt.Errorf("no CMS ID set for this ACO")
+	}
 	db := database.GetGORMDbConnection()
 	defer database.Close(db)
+	var cclfFile CCLFFile
+	// should I put a filter here to make sure it isn't too old?
+	if db.Debug().Where("aco_cms_id = ? and cclf_num = 8", aco.CMSID).Order("timestamp desc").First(&cclfFile).RecordNotFound() {
+		log.Errorf("Unable to find CCLF8 File for ACO: %v", *aco.CMSID)
+		return cclfBeneficiaryIDs, fmt.Errorf("unable to find cclfFile")
+	}
 
-	beneficiaryJoin := "inner join beneficiaries on acos_beneficiaries.beneficiary_id = beneficiaries.id"
-	if err = db.Table("acos_beneficiaries").Joins(beneficiaryJoin).Where("acos_beneficiaries.aco_id = ?", aco.UUID).Pluck("beneficiaries.blue_button_id", &beneficiaryIDs).Error; err != nil {
+	if err = db.Table("cclf_beneficiaries").Where("file_id = ?", cclfFile.ID).Pluck("ID", &cclfBeneficiaryIDs).Error; err != nil {
 		log.Errorf("Error retrieving ACO-beneficiaries for ACO ID %s: %s", aco.UUID.String(), err.Error())
```

Comment:
Might want to update the messaging here and below, since it's not just an ACO-beneficiary relationship anymore.
```suggestion
		log.Errorf("Error retrieving beneficiaries from latest CCLF8 file for ACO ID %s: %s", aco.UUID.String(), err.Error())
```